source/Mlos.Agent/MainAgent.cs

```diff
@@ -73,20 +73,26 @@ public void InitializeSharedChannel()
         {
             // Create or open the memory mapped files.
             //
+            Console.WriteLine("Create or open memory mapped files");

```

Comment:
The idea here is to give a little more feedback for the end to end example.

However, as noted in #104, we can't do this for production environments.

I think the intention is to proceed as is for the examples for now, and convert it some form of logger method (soon).

@grlap to comment further if need be.